runtimes/debian/rules

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --parallel
+
+override_dh_auto_configure:
+	cmake -S runtimes -B build_runtimes_genericjs \
+		-GNinja -C runtimes/CheerpCmakeConf.cmake \
```

Comment:
Like we discussed with Carlo the other day, we should add `-DCMAKE_TOOLCHAIN_FILE=/opt/cheerp/share/cmake/Modules/CheerpToolchain.cmake` here and `-DCMAKE_TOOLCHAIN_FILE=/opt/cheerp/share/cmake/Modules/CheerpWasmToolchain.cmake` for the wasm build

Reply:
Do you mean what I added on lines 10 and 14? Is there still something I should change?

Reply:
yes... not sure how I missed it